istioctl/cmd/proxyconfig.go

```diff
@@ -241,18 +285,28 @@ func proxyConfig() *cobra.Command {
 
   # Retrieve full cluster dump for clusters that are inbound with a FQDN of details.default.svc.cluster.local.
   istioctl proxy-config clusters <pod-name[.namespace]> --fqdn details.default.svc.cluster.local --direction inbound -o json
+
+  # Retrieve cluster summary without using Kubernetes API
+  curl <host>:15000/config_dump > envoy-config.json
```

Comment:
maybe a bit pedantic, but I think this doesn't work - the admin port is only on local host so <host> wouldn't work?

Reply:
done; it will now say `ssh <user@hostname> 'curl localhost:15000/config_dump' > envoy-config.json`